### PR TITLE
Update ros2-cookbooks commit

### PR DESCRIPTION
**Pull request by brawner**
This PR is for updating the ros2-cookbooks git submodule for https://github.com/ros-infrastructure/ros2-cookbooks/pull/15. This change will need to be updated once that PR is merged.

Windows Build
[Build Status](https://ci.ros2.org/job/ci_windows/12355/)

Signed-off-by: Stephen Brawner <brawner@gmail.com>

### CODE REVIEW

File: .gitmodules
Comment (jacobperron) [inline .gitmodules:22]: I guess this should remain as `latest`, assuming https://github.com/ros-infrastructure/ros2-cookbooks/pull/15 is merged. Right?

Comment (brawner) [reply .gitmodules:22]: Right, I'll update this PR when the ros2-cookbooks is merged. Still waiting for the CI job to complete

**Comment by brawner** [comment]
Running CI again after merging the upstream PR. The previous one failed due to what looks like a docker image issue.

[Build Status](https://ci.ros2.org/job/ci_windows/12356/)

**Review by jacobperron — APPROVED** [review]
LGTM pending CI

**Pull request merged**
